text/0000-api-jwt-authentication.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: default
+title: JWT Authentication for Fabric APIs
+nav_order: 3
+---
+
+- Feature Name: api_jwt_authentication
+- Start Date: 2021-01-11
+- RFC PR:
+- Fabric Component: orderer, peer
+- Fabric Issue:
+
+# Summary
+
+This document proposes an OAuth2 compatible authentication mechanism for MSP
+identities that produces authorization tokens suitable for use with APIs
+exposed by Fabric.
+
+# Motivation
+
+With the introduction of the channel participation API, the Fabric orderer
+exposes administrative operations that are authorized based on mutual TLS
+authentication. This approach was chosen as it protects the operations while
+maintaining conventional REST interaction patterns without explicit message
+hashing and signing.
+
+There are two minor issues with this approach:
+
+1. The client CA certificate used to control access to the channel
+   participation API is disconnected from the orderer's local MSP.
+2. It's often difficult for web applications to connect to servers with TLS
+   mutual authentication.
+
```

Comment:
The mutual TLS approach was known to be an interim solution.
I think it would be even more valuable to discuss the benefits relative to the traditional Fabric authentication approach (passing a client certificate in every interaction).